rllib/execution/train_ops.py

```diff
@@ -147,40 +179,10 @@ def multi_gpu_train_one_step(trainer, train_batch) -> Dict:
     trainer._counters[NUM_ENV_STEPS_TRAINED] += train_batch.count
     trainer._counters[NUM_AGENT_STEPS_TRAINED] += train_batch.agent_steps()
 
-    # Update weights - after learning on the local worker - on all remote
```

Comment:
Although I approve of this, this might break other training iteration functions currently in progress I think.

Reply:
Sorry, yes, we need to be careful with this. Moved this out, b/c we should separate concerns: train and update should be separate steps handled by separate functions.